### ISSUE TITLE
SVG (vector) Logo

### ISSUE THREAD

**Issue by karol-depka** [MichalLytek/type-graphql#824]
Hi.

It would be nice to provide a logo in SVG format.
Right now I only see PNG format.

If You have another format like .eps, I could do the conversion to .SVG.

Think of:
https://github.com/gilbarbara/logos.git 
:)

**Comment by MichalLytek** [comment]
@pkosiec is the logo creator but AFAIR he doesn't have the vector version 😞

**Comment by karol-depka** [comment]
Cześć, Hi, @MichalLytek, @pkosiec . Thanks. 

Can You give me whichever format You have, and I try to make SVG out of it?

**Comment by pkosiec** [comment]
Yes, I confirm that there's no SVG version of the logo. It was created in Pixelmator (`.pxm`) two years ago. Here's the file:
[logo.pxm.zip](https://github.com/MichalLytek/type-graphql/files/6175620/logo.pxm.zip)

However, as this project is way more popular as it was before, I would recommend getting the branding refresh from a professional designer (which I'm not). Hopefully it'll solve the issue 🙂

**Comment by karol-depka** [comment]
Thank You for such an extremely quick reply ! :).

Here is first attempt at SVG.
[logo.svg.zip](https://github.com/MichalLytek/type-graphql/files/6175631/logo.svg.zip)

**Comment by karol-depka** [comment]
[typegraphql-icon.svg.zip](https://github.com/MichalLytek/type-graphql/files/6175635/typegraphql-icon.svg.zip)
Just the logo (icon), no text. Transparent background.

**Comment by MichalLytek** [comment]
@karol-depka 
Thanks for your contribution! 🙏 

However, the colors are really inaccurate 😞 This can conflict with the theme colors in the docs site.

I would love to see a new logo, so if you would like to contribute in that way, I would be very greatful! 🙌

**Comment by karlbateman** [comment]
![typegraphql-logo](https://github.com/user-attachments/assets/58403e2b-344f-4d8a-a848-46f700057d54)

**Comment by karlbateman** [comment]
Sharing an alternative (with text) which merges Typescript and GraphQL but with a subtle difference to provide identity.

![typegraphql-logo-v2-text-LIGHT](https://github.com/user-attachments/assets/1587be8e-f00c-42e6-9865-c5f151e78b25)

And this is the logo.

![typegraphql-logo-v2](https://github.com/user-attachments/assets/58204bd4-63a5-4925-bc1e-c7e481c2067b)

Some notes on the design:

- Colour matched Typescript (`#3178C6`) and GraphQL (`#E10098`).
- Font is [Rubik Light](https://fonts.google.com/specimen/Rubik?sidebar.open=true&selection.family=Rubik:wght@300&preview.text=GraphQL&preview.text_type=custom) which has an [Open Font License](https://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL).
- Maintained the shadows used in the original logo for identity and to visually distinguish the two graph element colours.
- I have reversed the triangle to differentiate from GraphQL while providing familiarity.

All source files are attached.

[TypeGraphQL.zip](https://github.com/user-attachments/files/16936324/TypeGraphQL.zip)

Absolutely love this library, keep up the great work @MichalLytek ❤️ 🙌🏼

**Comment by karlbateman** [comment]
Sharing another alternative

![typegraphql-logo-v2-alternative](https://github.com/user-attachments/assets/ed61e0a8-09d3-4906-b687-da8225c272b2)

[typegraphql-logo-v2-alternative.svg.zip](https://github.com/user-attachments/files/16942332/typegraphql-logo-v2-alternative.svg.zip)
